docs/manage-fileset-metadata-using-gravitino.md

```diff
@@ -320,24 +320,41 @@ Currently, Gravitino supports two **types** of filesets:
 The `storageLocation` is the physical location of the fileset. Users can specify this location
 when creating a fileset, or follow the rules of the catalog/schema location if not specified.
 
+The `storageLocation` in each level can contain **placeholders**, format as `{{name}}`, which will
+be replaced by the corresponding fileset property value when the fileset object is created. The
+placeholder property in the fileset object is formed as "placeholder-{{name}}". For example, if
+the `storageLocation` is `file:///tmp/{{schema}}-{{fileset}}-{{verion}}`, and the fileset object 
+named "catalog1.schema1.fileset1" contains the properties `placeholder-version=v1`, 
+the actual `storageLocation` will be `file:///tmp/schema1-fileset1-v1`.
```

Comment:
Can you add a code example of how to use placeholders?

Reply:
added below